infra/recipes/terraform/azure/sandbox/vm-services.tf

```diff
@@ -0,0 +1,62 @@
+// define the virtual network
+resource "azurerm_virtual_network" "sandbox" {
```

Comment:
Assuming this vm configuration is for mysql. Can we use the same configuration for all the other services?

Reply:
Yes we should be able to. I only needed it for MySQL in this PR though

Reply:
Not following why this is connected to mysql? If we are using a managed mysql we should not be needing this special vm right? or is it part of mysql spinup?